frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -76,6 +76,9 @@ func (c *Controller) AddRoutes(router *mux.Router) {
 	// reached. Be prepared to retry if the deadline was reached.
 	router.Handle("/operations/{name}/:wait",
 		httpHandler(&waitOperationHandler{c.OperationManager, c.WaitOperationDuration})).Methods("POST")
+	router.Handle("/cvdbugreports",
```

Comment:
Despite the (misleading) command name, host bug reports in `cvd` are taken from a group, not the entire host. There should be a way to associate the bugreport to the group it was taken from, both at creation time and later. The best way to do this is to make the bugreports a subresource of `/cvds`

Reply:
Good point,

The new API looks like:

```
#  `:bugreport` is an operation of the cvd resource which creates the `cvdbugreport` resource.
POST /cvds/{group}/:bugreport

#  then we add new endpoints handling `cvdbugreport` as a main resource.
GET /cvdbugreport/{uuid}
DELETE /cvdbugreport/{uuid}
```